### ISSUE TITLE
Graphic bug when using Vulkan, OpenGL, D3D11 and Software in Ape Escape (Spain) (Instructions of Ski Kidz Racing)

### ISSUE THREAD

**Issue by edlink98** [libretro/swanstation#78]
**Description**
Here I report an error that is shown in the instructions of the Ski Kidz Racing mini game in the Ape Escape Spanish version. (Serial: SCES-02031) (Redump)
This graphic error appears when I use SwanStation with Vulkan, OpenGL, D3D11 and Software. (In Software mode the error is much more conspicuous.)
Beetle Psx HW with Vulkan and it is corrected when using OpenGL and Software. If I use the normal Beetle Psx it works perfectly. Already reported in their respective repositories. https://github.com/libretro/beetle-psx-libretro/issues/890

**Expected behavior**
While I was playing Ape Escape in the SwanStation core I was trying out the minigames and in Ski Kidz Racing and in the instructions there was an error on the right side of the screen.

Here I record what the error looks like when using each Renderer (Vulkan, OpenGL, D3D11 and Software). I also tested with Beetle PSX/HW but the error appears when using Vulkan. (Here I already report it in their repositories: https://github.com/libretro/beetle-psx-libretro/issues/890)

SwanStation: 3:48
Beetle PSX/HW: 0:00

VIDEO:
https://youtu.be/JW9H6JAgsEk

PHOTO:
![vlcsnap-2023-07-30-21h46m36s784](https://github.com/libretro/swanstation/assets/47231851/f74995cc-bd6e-45f0-8f62-0e2deefa61b6)

**Actual behavior**
This is how it should really look: https://youtu.be/JW9H6JAgsEk (0:55)

**Steps to reproduce the bug**

1. Use SwanStation with Vulkan, OpenGL, D3D11 or Software renderer.
2. Go to the minigames section on the left side to enter the first minigame.
3. Being inside the minigame you give Start - Inicio - Metodos

Bonus. Or just use this Save State:
https://drive.google.com/drive/folders/1POf3oWKqQs_jgNMdvJ6lyE0KUGRzaklm?usp=sharing

**Bisect results**
This started happening as soon as I noticed the error... I'm not sure how long it's been.

**Version/Commit**

-SwanStation: (1.0.0 27aad70)

**Core options**
The only modification I use is image scaling.
Internal Resolution Scale: 5x (1080p)
Multisample Antialiasing: off
Texture Filtering: Nearest-Neighbor
PGXP Geometry Correction: off

**Environment information**
Frontend: Retroarch 1.15.0 Stable
OS: Windows 10 Pro
GPU: Nvidia 2060 12gb (528.02)

**Comment by DarthMew** [comment]
Seems to only happen in (as far as I can test that is) the Spanish version, as the US version displays the screen just fine.
Also checked upstream, and it happens there as well.

**Comment by edlink98** [comment]
You are right in the US version it looks good even though something strange is seen at the top of the screen and I tried deactivating the Crop but it still continues. (I only tried it with Vulkan).

![Ape Escape (USA)-230731-164949](https://github.com/libretro/swanstation/assets/47231851/c2c84ffb-fe75-4f96-a0a8-e8b47e874405)
